chevah/compat/tests/normal/test_filesystem.py

```diff
@@ -101,15 +101,13 @@ def test_getPath_upper_paths(self):
         self.assertEqual(u'/c', path)
 
 
-class TestLocalFilesystem(CompatTestCase, FilesystemTestMixin):
+class DefaulFilesystemTestCase(CompatTestCase, FilesystemTestMixin):
```

Comment:
Curious why this is not `DefaultFilesystemTestCase`…

Reply:
typo :) Will Fix. Thanks!